lessons/Early-Math/identity-property/langs/en-us/index.adoc

```diff
@@ -167,33 +167,104 @@ Students explore two functions - `scale` and `rotate` - each of which consumes a
 
 === Launch
 
-Let's say we have a number, `w`. The Identity Property tells us that we can multiply or divide `w` by 1... and we'll always get `w` back. If we multiply `w` by some value *other* than 1, however, we should expect a new outcome!
+You’ve discovered that multiplying or dividing an expression by 1 does not change its value. Similarly, adding or subtracting 0 results in the original value. Each of these four applications of the Identity Property is represented in the four Circles of Evaluation below.
+
+@ifnotslide{
+[.embedded, cols="^.^6a,^.^6a,^.^6a,^.^6a",stripes="none"]
+|===
+
+| @show{(coe '(* m 1))}
+| @show{(coe '(/ m 1))}
+| @show{(coe '(+ m 0))}
+| @show{(coe '(- m 0))}
+
+|===
+}
+
+Each of these four Circles evaluates to @math{m} which means *the Identity Property requires an operation that does… nothing!*
```

Comment:
I LOVE this table and the summary statement! If it were the late 90s I would demand that you make a Seinfeld reference "It's arithmetic about _nothing!_"

I feel like this belongs in the Synthesize section of "The Identity Property and Variables." Besides, didn't we already agree that one-input identity functions that work with numbers belong firmly in the _non-programming_ section?

Reply:
we previously talked about one-input functions (i.e., triple) with respect to the distributive property, not the identity property.... but it seems like this is a recurring theme! I like this feedback.